src/diffusers/pipelines/cogvideo/pipeline_cogvideox.py

```diff
@@ -446,19 +446,34 @@ def _prepare_rotary_positional_embeddings(
         p = self.transformer.config.patch_size
         p_t = self.transformer.config.patch_size_t or 1
 
-        base_size_width = self.transformer.config.sample_width // p
-        base_size_height = self.transformer.config.sample_height // p
-        base_num_frames = (num_frames + p_t - 1) // p_t
+        if p_t is None:
+            # CogVideoX 1.0
+            base_size_width = self.transformer.config.sample_width // p
+            base_size_height = self.transformer.config.sample_height // p
```

Comment:
This part is the same if `p_t == None` or not right? Can't we just move it outside the if/else so the lines aren't duplicated?

Reply:
Oh yep, moving it out now